export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
If I'm interpreting my own code right:

If the material has a node tree it will run through all the nodes.  If it finds a bake texture node it'll return True and end.  If it doesn't it'll return False.  If the material doesn't have a node tree it'll return True and end.